generators/chipyard/src/main/scala/config/HeteroConfigs.scala

```diff
@@ -99,30 +71,6 @@ class LargeBoomAndHwachaRocketConfig extends Config(
 // DOC include end: DualBoomAndRocketOneHwacha
 
 
-
-class LargeBoomAndRV32RocketConfig extends Config(
```

Comment:
Why delete this config?

Reply:
Also I don't think this config does what it describes. I'm pretty sure it instantiates 2x RV64 booms and 1x RV64 rocket